tokumx/tox.ini

```
@@ -1,12 +1,13 @@
 [tox]
 minversion = 2.0
-basepython = py27
+basepython = py37
```

Since no py3 support
```suggestion
basepython = py27
```